### PR TITLE
free, lsblk: add pt_BR translation

### PR DESCRIPTION
**Pull request by Fabiokleis**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/linux/lsblk.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- Lista todos dispositivos de armazenamento no formato de árvore:
+
+`lsblk`
+
+- Também lista dispositivos vazios:
+
+`lsblk -a`
+
+- Mostra a coluna de tamanhos em bytes, em vez de um formato legível por humanos:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/lsblk.md:14]: I don't speak pt_BR, but I think this is the correct verb:
```suggestion
- Mostrar a coluna de tamanhos em bytes, em vez de um formato legível por humanos:
```

**Review by navarroaxel — COMMENTED** [review]
Hi @Fabiokleis, welcome and thanks for your submission. I left a comment.

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@Fabiokleis could you please move the new `pacstall` page to a new PR?

**Comment by Fabiokleis** [comment]
@navarroaxel ok I'll move to a new one

**Comment by navarroaxel** [comment]
@Fabiokleis this is why you shouldn't create PRs from your `main` :upside_down_face: Let me know if you need help to fix this.

**Comment by Fabiokleis** [comment]
@navarroaxel I just realized I did it wrong, if you could help me... ushuahsuahu

**Comment by navarroaxel** [comment]
Right now you should create a branch from your `main` `git checkout -b pacstall`.
Then you should go back to main and push-force your translations to leave this PR in the previous state.
After merge, you can `reset` your `main` to leave with the same history as the `tldr-pages:main`.

**Comment by Fabiokleis** [comment]
@navarroaxel  I reset main to my last commit of translation, now I need to create a new PR, but how to do this without change the main ?

**Comment by navarroaxel** [comment]
@Fabiokleis you have to create a new branch from `main` and then write the correct history to merge than branch to the `tldr-pages:main`. When you create a PR across forks you can select the any branch from your fork as `source branch`.

**Comment by Fabiokleis** [comment]
@navarroaxel I created a new branch and create a new PR, now I think that its all right, thank you a lot.

**Review by CleanMachine1 — APPROVED** [review]

**Comment by navarroaxel** [comment]
@Fabiokleis merged :raised_hands: now try to sync your `main` with `tldr-pages:main`.

**Pull request merged**